commands_test.go

```diff
@@ -91,7 +91,7 @@ func TestGetCommands(t *testing.T) {
 	c.Context = ctx
 
 	commands := getCommands(ctx, act, app)
-	assert.Equal(t, 34, len(commands))
+	assert.Equal(t, 32, len(commands))
```

Comment:
Hooray! :-)

Reply:
> Where to place YAML / KV support of the secrets. Is that provided by the API?

Yes, this will be part of the API.
However that part is not fully fleshed out, yet.
So far this is still exposing the old secrets interface.

After this PR has been merged I'd like to work on #1310 which will most likely overhaul the secrets implementation and change the corresponding API.

> As long as we refactor out CLI commands in the same repo, I things we're in a good shape of experimenting with the API without breaking it. Ideally, we could also try to do the same with gopass main command.

No, I don't plan to have the main gopass binary use the API ever. At least not before a 2.0 release. Maybe after that, but probably not. The reason is simple: the main binary needs to access a lot of internals which are going to change a lot. This would make the API ugly, brittle and overly complicated. Even after we introduce an API gopass will still primarily be an CLI programm. The API is just a very simple and intentionally limited extension point for features we don't want to merge in the main binary.

We can certainly revisit this discussion after we have a stable 2.x release with a trimmed down feature set, but right now this would be a lot of extra work for little or no benefit.